R/derive_var_analysis_ratio.R

```diff
@@ -0,0 +1,81 @@
+#' Derive Ratio Variable
+#'
+#' Derives a ratio variable for a BDS dataset. The default behavior will take
+#' denominator variable and prefix it with `"R2"`. For example, a user wishing to
+#' calculate a Ratio to Baseline, `"AVAL / BASE"` will have returned a new
+#' variable `"R2BASE`. Ratio to Analysis Range Lower Limit `"AVAL / ANRLO"`
+#' will return a new variable `"R2ANRLO"`, and Ratio to Analysis Range  Upper
+#' Limit `"AVAL / ANRHI"` will return a new variable `"R2ANRLO"`. However, a user
```

Comment:
Double quotes should not be used for variable names and expressions.

I would move the part describing the default behavior of `new_var` to the description of the `new_var` parameter.

Here or in the details section should be mentioned that `NA` is returned if the denominator is zero because this deviates from the standard R behavior.